data/api/v1/full_spec.yaml

```diff
@@ -14185,6 +14202,46 @@ components:
           format: double
           type: number
       type: object
+    SyntheticsTestOptionsScheduling:
+      description: Object containing timeframes and timezone used for advanced scheduling.
+      properties:
+        timeframes:
+          description: Array containing objects describing the scheduling pattern
+            to apply to each day.
+          example:
+          - day: 1
+            from: 07:00
+            to: '16:00'
+          - day: 3
+            from: 07:00
+            to: '16:00'
+          items:
+            $ref: '#/components/schemas/SyntheticsTestOptionsSchedulingTimeframe'
+          type: array
+        timezone:
+          description: Timezone on which the timeframe are based.
```

Comment:
```suggestion
          description: Timezone in which the timeframe is based.
```

Reply:
This is actually part of an unrelated PR, which got rolled into this one by mistake. I've opened another PR on the spec to fix this - https://github.com/DataDog/datadog-api-spec/pull/2020